modules/local/format_helitronscanner/main.nf

```diff
@@ -0,0 +1,55 @@
+process FORMAT_HELITRONSCANNER_OUT {
```

Comment:
Let's rename the directory to `format_helitronscanner_out` for the sake of consistency.